packages/grafana-data/src/types/plugin.ts

```diff
@@ -192,14 +184,6 @@ export class GrafanaPlugin<T extends PluginMeta = PluginMeta> {
     return this;
   }
 
-  /**
-   * Specify how the plugin should support paths within the live streaming environment
-   */
-  setChannelSupport(support: LiveChannelSupport) {
```

Comment:
@mckn -- perhaps we should leave this function with a signature like:

```
setChannelSupport(support: any) {
  console.warn(...);
}
```

Reply:
hmm, is it being replaced by some other API? I like the idea of having a `console.warn(..)`. Also it would make sense to add the `@deprecation` attribute in the code comments.

Reply:
Also, we should add a section and some information in the migration guide for this:

https://github.com/grafana/grafana/blob/main/docs/sources/developers/plugins/migration-guide.md

Reply:
I'm not sure we should add it to a migration guide -- this was both never used (except to fill ui dropdown in internal plugins) and never published.  All the types have been @alpha -- and comments for experimental.  

Current code is:
```

  /**
   * @deprecated -- this is no longer necessary and will be removed
   */
  setChannelSupport(support: any) {
    console.warn('[deprecation] plugin is using ignored option: setChannelSupport', this.meta);
    return this;
  }
  ```